check-disk/lib/check-disk.go

```diff
@@ -75,6 +64,29 @@ func checkStatus(current checkers.Status, threshold string, units float64, disk
 	return current, nil
 }
 
+func checkInodes(current checkers.Status, threshold string, disk *gpud.UsageStat, status checkers.Status) (checkers.Status, error) {
+	if !strings.HasSuffix(threshold, "%") {
+		return checkers.UNKNOWN, errors.New("-W, -K value should be N%")
+	}
+
+	thresholdPct, err := strconv.ParseFloat(strings.TrimRight(threshold, "%"), 64)
+	if err != nil {
+		return checkers.UNKNOWN, err
+	}
+
+	// Skip checking if disk.InodesTotal == 0 since inodesFreePct is meaningless.
+	if disk.InodesTotal == 0 {
+		return current, nil
```

Comment:
I kept the previous behavior (just skipping checks), but perhaps it would be better to return UNKNOWN status.

Reply:
What do you think? @astj